test/test_reynir.py

```diff
@@ -351,6 +351,192 @@ def test_lemmas():
+    assert s.lemmas == ['hér', 'vera', 'Svanur Hildar- og Pálsson Scheving', '.']
+
+    s = g.parse_single("Hér eru Áki og Andri Brjánssynir.")
+    #assert s.lemmas == ['hér', 'vera', 'Áki og Andri Brjánssynir', '.']    # Out of scope 
+
+    s = g.parse_single("Hér eru Ína og Una Brjánsdætur.")
+    #assert s.lemmas == ['hér', 'vera', 'Ína og Una Brjánsdætur', '.']    # Out of scope 
+
+    s = g.parse_single("Hér eru Áki og Láki Brjánssynir.")
+    #assert s.lemmas == ['hér', 'vera', 'Áki og Láki Brjánssynir', '.']    # Out of scope 
+
+    s = g.parse_single("Hér eru Ína og Mína Brjánsdætur.")
+    #assert s.lemmas == ['hér', 'vera', 'Ína og Mína Brjánsdætur', '.']    # Out of scope 
+
+    s = g.parse_single("Hér eru Áki og Ína Brjánsbörn.")
+    #assert s.lemmas == ['hér', 'vera', 'Áki og Ína Brjánsbörn', '.']    # Out of scope 
+
+    s = g.parse_single("Hér er Jack Nicholson.")
+    assert s.lemmas == ['hér', 'vera', 'Jack Nicholson', '.']
+
+    s = g.parse_single("Hér er Diane Lane.")
+    assert s.lemmas == ['hér', 'vera', 'Diane Lane', '.']
+
+    s = g.parse_single("Hér er Finsbury Park.")
+    assert s.lemmas == ['hér', 'vera', 'Finsbury Park', '.']
+
+    s = g.parse_single("Hér er Sky Sports.")
+    assert s.lemmas == ['hér', 'vera', 'Sky Sports', '.']
+
+    s = g.parse_single("Hér er J. K. Rowling.")
+    #assert s.lemmas == ['hér', 'vera', 'J. K. Rowling', '.']    # Out of scope 
+
+    s = g.parse_single("Hér er Parsley Ecothelial Welmington III.")
+    assert s.lemmas == ['hér', 'vera', 'Parsley Ecothelial Welmington III', '.']
+
+    s = g.parse_single("Hér er Dietrich van Helsing.")
+    assert s.lemmas == ['hér', 'vera', 'Dietrich van Helsing', '.']
+
+    s = g.parse_single("Hér er Helmine van de Fnupft.")
+    assert s.lemmas == ['hér', 'vera', 'Helmine van de Fnupft', '.']
+
+    s = g.parse_single("Hér er Carla de la Cruz.")
+    assert s.lemmas == ['hér', 'vera', 'Carla de la Cruz', '.']
+
+    s = g.parse_single("Hér er Barack Obama.")
+    assert s.lemmas == ['hér', 'vera', 'Barack Obama', '.']
+
+    s = g.parse_single("Hér er Finnur de la Cruz.")
+    assert s.lemmas == ['hér', 'vera', 'Finnur de la Cruz', '.']
+
+    s = g.parse_single("Hér er Derek Árnason.")
+    #assert s.lemmas == ['hér', 'vera', 'Derek Árnason', '.']
+
+    s = g.parse_single("Hér er Díana Woodward.")
+    assert s.lemmas == ['hér', 'vera', 'Díana Woodward', '.']
+
+    s = g.parse_single("Hér er Knut Axel Holding AS.")
+    #assert s.lemmas == ['hér', 'vera', 'Knut Axel Holding AS', '.']    # Out of scope 
+
+    s = g.parse_single("Hér er Matthildur Ármannsdóttir ehf..")
+    #assert s.lemmas == ['hér', 'vera', 'Matthildur Ármannsdóttir ehf.', '.']   # Out of scope
+
+    s = g.parse_single("Hér er Super Mattel AS.")
+    assert s.lemmas == ['hér', 'vera', 'Super Mattel AS', '.']
+
+    s = g.parse_single("Hér er WOW Cyclothon.")
+    #assert s.lemmas == ['hér', 'vera', 'WOW Cyclothon', '.']   # Out of scope
```

Comment:
Why out of scope?

Reply:
We don't currently combine acronyms and entity names. I include the test in case we want to add that later.